packages/eslint-plugin-circuit-ui/prefer-custom-properties/README.md

```diff
@@ -0,0 +1,46 @@
+# Do not use invalid Circuit UI custom properties (`prefer-custom-properties`)
+
+Circuit UI's Emotion.js theme is being replaced with CSS custom properties (a.k.a. CSS variables). This rule flags uses of the Emotion.js theme and can automatically rewrite them to CSS custom properties.
+
+## Rule Details
+
+This rule aims to automate the migration to CSS custom properties. CSS custom properties are widely supported and more performant to update than Emotion.js' theme object that is passed to components via React context. Setting the rule's error level to `warn` (or `1`) is recommended.
+
+Examples of **incorrect** code for this rule:
+
+```ts
+const styles = (theme) => css`
+  padding: ${theme.spacings.kilo};
+`;
+
+const Box = styled.div`
+  padding: ${(theme) => theme.spacings.kilo};
+`;
+```
+
+Examples of **correct** code for this rule:
+
+```ts
+const styles = () => css`
+  padding: var(--cui-spacings-kilo);
+`;
+
+// CSS custom properties aren't supported inside media queries
```

Comment:
I'm not sure I follow here; it says this is correct code, but here it says is not supported... 🤔

Reply:
The code below uses the legacy JavaScript theme, which gets replaced by a hard-coded media query. I added this to explain why the ESLint rule doesn't complain and replace the media query tokens with CSS custom variables. I'll rephrase the comment to clarify.